azure-ts-serverless-url-shortener-global/index.ts

```diff
@@ -0,0 +1,120 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+import { Container } from "@azure/cosmos";
+import { getContainer } from "./cosmosclient";
+
+// Read a list of target locations from the config file:
+// Expecting a comma-separated list, e.g., "westus,eastus,westeurope"
+const config = new pulumi.Config();
+const locations = config.require("locations").split(',');
+// The first location is considered primary
+const primaryLocation = locations[0];
+
+let resourceGroup = new azure.core.ResourceGroup("urlshort-rg", {
```

Comment:
is `-rg` a common azure naming strategy?  if so, this is fine.  But otherwise, i think we'd generally prefer the names not contain these sorts of hungarian notations.

Reply:
No, it's not common. I'll try to find better names.